.travis.yml

```diff
@@ -30,11 +30,11 @@ matrix:
     - os: linux
       dist: trusty
       sudo: required
-      env: DOCKER_IMAGE=ubuntu:16.04 KITURA_NIO=1 TESTDB_NAME=testdb_1604_1
+      env: DOCKER_IMAGE=ubuntu:16.04 KITURA_NIO=1 DOCKER_ENVIRONMENT="TESTDB_NAME" TESTDB_NAME=testdb_1604_1
```

Comment:
It's surprising to me that this isn't `DOCKER_ENVIRONMENT="TESTDB_NAME=testdb_1604_1"`